internal/usage/usage.go

```diff
@@ -226,6 +226,7 @@ const (
 	NetworkAccessListIPEntries      = "List of IP addresses to be allowed to access the deployment."
 	NetworkAccessListIPEntry        = "IP address to be allowed to access the deployment."
 	AccessListCIDREntry             = "Whitelist entry in CIDR notation to be added for a given API key."
+	AccessListCIDREntries           = "IP address access list entries associated with the link token."
```

Comment:
let's give this a live migration/link token related name, it tends to happen that people will copy/paste flag definitions without reading the copy and then wrong copy gets applied to the wrong command

Reply:
Good idea, will do.